### PR TITLE
Website updates

### PR DESCRIPTION
**Pull request by inexorableAce**
Updated the images and some outdated copy on the website, things like replacing trials with gambit, bones of eao, with STOMP-EE5, etc. Also updated the colors to fit the style guide we created and added some hover over interaction to the launch app buttons.

### CODE REVIEW

**Comment by bhollis** [comment]
Screenshots?

File: destinyitemmanager.com/index.html [hunk 1]
```diff
@@ -1,167 +1,177 @@
...
+  </section>
+  <section class="easy">
+    <div class="row">
+      <div class="col-5 image"><img class="image-shadow" alt="Inspect Weapons" src="images/InspectWeaponShot@2x.jpg"
+          width="379px"></div>
+      <div class="col-5">
+        <div class="text-wrap">
+          <h2>It Really Is That Easy.</h2>
+          <h3>Drop Rating Percentage</h3>
+          <p>Going for a Tier 12 build? Use DIM to see how good your roll is on armor. (Coming soon for Destiny 2)</p>
```
Comment (bhollis) [inline destinyitemmanager.com/index.html]: We should probably remove this.

File: destinyitemmanager.com/index.html [hunk 2]
```diff
@@ -1,167 +1,177 @@
...
-</html>
+    </div>
+    <div class="row">
+      <div class="col-6">
+        <ul class="footer-bottom-nav">
+          <li>Destiny is a registered trademark of Bungie. Some content and images are the property of Bungie.</li>
+        </ul>
+      </div>
+      <div class="col-6">
+        <div class="copy">&copy; 2017 DIM</div>
```
Comment (bhollis) [inline destinyitemmanager.com/index.html]: 2018!

File: destinyitemmanager.com/index.html [hunk 3]
```diff
@@ -1,167 +1,177 @@
...
+    </div>
+  </section>
+  <section class="easy">
+    <div class="row">
+      <div class="col-5 image"><img class="image-shadow" alt="Inspect Weapons" src="images/InspectWeaponShot@2x.jpg"
+          width="379px"></div>
+      <div class="col-5">
+        <div class="text-wrap">
+          <h2>It Really Is That Easy.</h2>
+          <h3>Drop Rating Percentage</h3>
```
Comment (bhollis) [inline destinyitemmanager.com/index.html]: We could add search as a top level bullet

File: destinyitemmanager.com/index.html (hunk 2 above)
Comment (inexorableAce) [reply destinyitemmanager.com/index.html]: I wasn't sure about that, do you change the year every year on copyright? or is it the year it was made?

File: destinyitemmanager.com/index.html (hunk 1 above)
Comment (inexorableAce) [reply destinyitemmanager.com/index.html]: Yeah also wasn't sure here, I know its really close to being in beta so I changed the wording to coming soon.

**Comment by inexorableAce** [comment]
Screenshots.
image
image
image
image

File: destinyitemmanager.com/index.html (hunk 3 above)
Comment (inexorableAce) [reply destinyitemmanager.com/index.html]: yeah replace t12 with search? sounds good.

**Comment by bhollis** [comment]
Love it!

**Pull request merged**